web/grab_to_pan.js

```diff
@@ -176,21 +178,27 @@ GrabToPan.prototype = {
     // Note: ChildNode.remove doesn't throw if the parentNode is undefined.
     this.overlay.remove();
   },
-};
 
-// Get the correct (vendor-prefixed) name of the matches method.
-let matchesSelector;
-["webkitM", "mozM", "m"].some(function (prefix) {
-  let name = prefix + "atches";
-  if (name in document.documentElement) {
-    matchesSelector = name;
-  }
-  name += "Selector";
-  if (name in document.documentElement) {
-    matchesSelector = name;
-  }
-  return matchesSelector; // If found, then truthy, and [].some() ends.
-});
+  /**
+   * @private
+   */
+  get _matchesSelector() {
```

Comment:
We *could* perhaps even add a fast-path for MOZCENTRAL here, e.g. as outlined below, however given that this code no longer runs *unconditionally* that's probably not necessary.
```javascript
if (typeof PDFJSDev !== "undefined" && PDFJSDev.test("MOZCENTRAL")) {
  return shadow(this, "matchesSelector", "matches");
}
```